packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -142,13 +139,37 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
         aria-label={ link.ariaLabel }
         role='tab'
         aria-selected={ this.state.selectedKey === itemKey }>
-        { icon }
-        { text }
-        { count }
+        { linkContent }
       </button>
     );
   }
 
+  @autobind
+  private _renderLinkContent(link: IPivotItemProps): JSX.Element {
+    const { itemCount, itemIcon, linkText } = link;
+
+    let count: JSX.Element;
+    if (itemCount !== undefined) {
+      count = <span className='ms-Pivot-count'>({ itemCount })</span>;
+    }
+
+    let icon: JSX.Element;
+    if (itemIcon !== undefined) {
+      icon = <span className='ms-Pivot-icon'><i className={ `ms-Icon ms-Icon--${itemIcon}` }></i></span>;
+    }
+
+    let text: JSX.Element;
+    if (linkText !== undefined) {
+      text = <span className='ms-Pivot-text'>{ link.linkText }</span>;
+    }
+
+    return <span className='ms-Pivot-link-content'>
+      { icon }
+      { text }
+      { count }
```

Comment:
can we do these inline 
`{ itemCount && <span className='ms-Pivot-count'>({ itemCount })</span> }` ?. I find it easier to read JSX that way. Not pushing for it though.